packages/core/content-manager/server/src/services/document-manager.ts

```diff
@@ -193,44 +193,27 @@ const documentManager = ({ strapi }: { strapi: Core.Strapi }) => {
       return strapi
         .documents(uid)
         .publish({ ...params, documentId: id })
-        .then((result) => result?.versions.at(0));
+        .then((result) => result?.versions);
+      // TODO __** check this return format change okay?
     },
 
-    async publishMany(entities: Document[], uid: UID.ContentType) {
-      if (!entities.length) {
-        return null;
-      }
-
-      // Validate entities before publishing, throw if invalid
-      await Promise.all(
-        entities.map((document: Document) => {
-          return strapi.entityValidator.validateEntityCreation(
-            strapi.getModel(uid),
-            document,
-            undefined,
-            // @ts-expect-error - FIXME: entity here is unnecessary
-            document
-          );
-        })
-      );
+    async publishMany(uid: UID.ContentType, documentIds: string[], locale?: string | string[]) {
+      let publishedEntitiesCount = 0;
+      await async.map(documentIds, async (documentId: string) => {
```

Comment:
this work might collide with @Feranchz , he is working on bulk publsihing

Reply:
Yes, but this PR is more advanced than mine and in any case I was doing something **very** similar so maybe we can go ahead with this. 

I will change my PR's scope to only the bulkDelete which is not changed in this PR and follow this